src/coreclr/nativeaot/BuildIntegration/Microsoft.NETCore.Native.Unix.targets

```diff
@@ -74,8 +76,16 @@ The .NET Foundation licenses this file to you under the MIT license.
       <NativeFramework Include="GSS" />
     </ItemGroup>
 
+    <ItemGroup>
```

Comment:
`<IlcPath>` should point to the package root. Can you use that instead of trying to guess it here?

Reply:
I used <IlcSdkPath> in next revision

Reply:
I hope using `IlcSdkPath` is good

Reply:
Was there a problem with `IlcPath`? It would make the `..` go away.